flow/include/flow/BlobCipher.h

```diff
@@ -396,8 +396,8 @@ class BlobCipherKeyCache : NonCopyable, public ReferenceCounted<BlobCipherKeyCac
 	                                         int baseCipherLen,
 	                                         const EncryptCipherRandomSalt& salt);
 
-	// API returns the last insert cipherKey for a given encryption domain Id.
-	// If domain Id is invalid, it would throw 'encrypt_invalid_id' exception,
+	// API returns the last insert cipherKey for a given encryption domain id.
+	// If domain id is invalid, it would throw 'encrypt_invalid_id' exception,
```

Comment:
```suggestion
	// API returns the last insert cipherKey for a given encryption domain Id.
	// If domain Id is invalid, it would throw 'encrypt_invalid_id' exception,
```